modules/gapi/include/opencv2/gapi/s11n.hpp

```diff
@@ -17,72 +17,159 @@
 namespace cv {
 namespace gapi {
 
+/**
+* \addtogroup gapi_serialization
+* @{
+*/
+
 namespace detail {
```

Comment:
did you check that functions were not listed before adding `@private`?

Rebase to the latest `master`, this namespace should be entirely excluded from the documentation already

Reply:
Done